jupyterlab_latex/__init__.py

```diff
@@ -117,13 +119,19 @@ def build_tex_cmd_sequence(self, tex_base_name, run_bibtex=False):
             escape_flag = '-no-shell-escape'
         elif c.shell_escape == 'restricted':
             escape_flag = '-shell-restricted'
-            
+
+        # Get the synctex query parameter, defaulting to
+        # 1 if it is not set or is invalid.
```

Comment:
why are you defaulting to on? I would default to off as it produces extra stuff. Or am i misreading this?

Reply:
You are reading correctly. I thought that it was nicer to default to on, since it's a useful functionality. It produces another file, but I didn't view that as a big downside.

Reply:
well… in that case, I'd prefer that the whitelist not include synctex unless it's explicitly requested… because I'd be really annoyed if I were to run this and somehow it created unexpected files…

Reply:
We are not creating a `.synctex.gz` file if it is not explicitly requested, because we are setting `-synctex=0`, so keeping it in the whitelist shouldn't really have an effect.